src/api/migrations/20190625141142-create-reasons.js

```diff
@@ -0,0 +1,22 @@
+export default {
+  up: (queryInterface, Sequelize) => queryInterface.createTable('Reasons', {
+    id: {
+      allowNull: false,
+      autoIncrement: true,
+      primaryKey: true,
+      type: Sequelize.INTEGER
+    },
+    name: {
```

Comment:
what are you storing as a name? @MCFrank16

Reply:
that is the reason name @truestbyheart